integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/testsupport/configuration/AnalysisOverrideITAnalysisConfigurer.java

```diff
@@ -21,5 +22,14 @@ public void configure(ElasticsearchAnalysisDefinitionContainerContext context) {
 		context.analyzer( OverrideAnalysisDefinitions.ANALYZER_WHITESPACE_LOWERCASE.name ).custom()
 				.withTokenizer( "whitespace" )
 				.withTokenFilters( "lowercase" );
+
+		String tokenizerName = AnalysisCustomIT.AnalysisDefinitions.ANALYZER_PATTERNS_STOPWORD.name + "_tokenizer";
```

Comment:
Please don't do that: `AnalysisCustomIT.AnalysisDefinitions` is completely unrelated. Just use `OverrideAnalysisDefinitions.ANALYZER_NGRAM.name + "_tokenizer`?

Reply:
Yep! It's copy and paste error.